bridgev2/matrix/intent.go

```diff
@@ -674,3 +674,23 @@ func (as *ASIntent) MuteRoom(ctx context.Context, roomID id.RoomID, until time.T
 		})
 	}
 }
+
+func (as *ASIntent) GetEvent(ctx context.Context, roomID id.RoomID, eventID id.EventID) (*event.Event, error) {
+	evt, err := as.Matrix.Client.GetEvent(ctx, roomID, eventID)
+	if err != nil {
+		return nil, err
+	}
+	err = evt.Content.ParseRaw(evt.Type)
+	if err != nil {
+		zerolog.Ctx(ctx).Err(err).Str("roomID", roomID.String()).Str("eventID", string(eventID)).Msg("failed to parse event content")
```

Comment:
`Stringer(x)` is preferred over `Str(x.String())` (I just noticed the lint rule wasn't enabled, so enabled it on main just now)